test/snippet/range/view/take_exactly.cpp

```diff
@@ -1,21 +1,18 @@
+#include <string>
+
 #include <string>
 #include <seqan3/core/debug_stream.hpp>
 #include <seqan3/range/view/take_exactly.hpp>           // provides view::take_exactly and view::take_exactly_or_throw
 
-using namespace seqan3;
-
 int main()
 {
     std::string vec{"foobar"};
-    auto v = vec | view::take_exactly(3);               // or view::take_exactly_or_throw
-    debug_stream << v << '\n';                          // "foo"
-    debug_stream << std::ranges::size(v) << '\n';       // 3
-
+    auto v = vec | seqan3::view::take_exactly(3);               // or seqan3::view::take_exactly_or_throw
+    seqan3::debug_stream << v << '\n';                          // "foo"
+    seqan3::debug_stream << std::ranges::size(v) << '\n';       // 3
 
-    auto v2 = vec | view::take_exactly(9);
-    debug_stream << std::ranges::size(v2) << '\n';      // 9 <- here be dragons!
-//     debug_stream << v2 << '\n';                      // possibly memory access violation
 
-//     auto v3 = vec | view::take_exactly_or_throw(9);  // safe, throws immediately on construction
+    auto v2 = vec | seqan3::view::take_exactly(9);
```

Comment:
I think this must be out commented as this throws and the test will likely fail

Reply:
I think the test actually passes, it just has some random number in there.